src/components/Button/Button.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+import styles from "./Button.module.css";
+
+function Button({ children, isButtonActive, ...props }) {
+  return (
+    <button
+      className={`${styles.Button} ${isButtonActive ? styles.active : ""}`}
```

Comment:
`styles.Button`
`styles.active`

tutarsız görünüyor. Baş harflerinin büyük olması daha iyidir fakat hepsinin tutarlı olması bundan daha önemli. Bu sayede koda devam ettiğin zaman bir ön görü geliştirirsin. Küçük harfle miydi ya da büyük harfle miydi gibi soruların oluşmaz.

Ek olarak `styles`  için `active` demeyi yeterli bulmuşken `props` olarak buton'a `isButtonActive` demekte kendi içinde biraz tutarsız. Şu yorumda bu konuya değiniyor.

https://github.com/gizemnkorkmaz/pomodoro-timer/pull/3/files#r795400768